src/resources/person.py

```diff
@@ -21,18 +22,63 @@ def get(self):
         Returns one or more user objects
 
         Params:
-            email   User's email address
-            fname   First name of the user
-            lname   Last name of the user
-            comyear Community year of the user
+            email       User's email address
+            fName       First name of the user
+            lName       Last name of the user
+            comYearMIN  Minimum community year of the user (inclusive)
+            comYearMAX  Maximum community year of the user (inclusive)
         """
-        raise NotImplementedError
+        """
+        we pull everything with db.query(all)? 
+        Or query = Person.objects() and then 
+        if params.get('fName'):
+            query = query.filter(fName = params['fName'])
+        if params.get('comYearMIN'):
+            query = query.filter(comYear > int(params['comYearMIN']))
+
+        and then at the end:
+        return [person.to_json() for person in query]
+
+        """
+        params = request.args
+        query = Person.objects()
+        if params.get('fName'):
+            query = query.filter(fName = params['fName'])
+        if params.get('lName'):
+            query = query.filter(lName = params['lName'])
+        if params.get('email'):
+            query = query.filter(email = params['email'])
+        if params.get('comYearMIN'):
+            query = query.filter(comYear__gte = int(params['comYearMIN']))
+        if params.get('comYearMAX'):
+            query = query.filter(comYear__lte = int(params['comYearMAX']))
+        return [person.to_json() for person in query]
 
     def put(self):
-        raise NotImplementedError
+        """
+        If requests.json's fields do not match those defined in the Person model, this fails. 
+        See documentation for src.document_models.Person for more details.
+        """
+        object = Person(**request.json) 
```

Comment:
we'll want to put better error handling in here so that the user gets a message like `field ___ was missing, must be integer` or something like that - so that you have some help getting things fixed